LiteCore/Query/N1QL_Parser/n1ql.leg

```diff
@@ -423,10 +434,12 @@ SELECT    = "SELECT"i      WB
 SOME      = "SOME"i        WB
 THEN      = "THEN"i        WB
 TRUE      = "TRUE"i        WB
+UNNEST    = "UNNEST"i      WB
 VALUED    = "VALUED"i      WB
 WHEN      = "WHEN"i        WB
 WHERE     = "WHERE"i       WB
 
+reservedWord = ALL | AND | ANY | AS | ASC | BETWEEN | BY | CASE | COLLATE | CROSS | DESC | DISTINCT | ELSE | END | EVERY | EXISTS | FALSE | FROM | GROUP | HAVING | IN | INNER | IS | JOIN | LEFT | LIKE | LIMIT | MISSING | NOT | NULL | OFFSET | ON | OR | ORDER | OUTER | SATISFIES | SELECT | SOME | THEN | TRUE | UNNEST | VALUED | WHEN | WHERE
```

Comment:
Will this cause any performance impact to the parser?

Reply:
It should be performant because, I think, LEG compiler will create a regex state machine. In any case, the performance impact will be minuscule, unless dealing with large n1ql file with a lot of identifiers.